exporters/secret/error.go

```diff
@@ -0,0 +1,14 @@
+package secret
+
+import (
+	"github.com/giantswarm/microerror"
+)
+
+var certNotFoundError = &microerror.Error{
+	Kind: "certNotFoundError",
+}
+
+// NoCertFound asserts cerNotFoundError
+func NoCertFound(err error) bool {
```

Comment:
```suggestion
func IsCertNotFound(err error) bool {
```

Reply:
Those functions usually start with `Is` and contain the error name ([link](https://github.com/giantswarm/fmt/blob/master/go/errors.md#custom-errors))

Reply:
Thanks for the `fmt` link - very helpful :)